### PR TITLE
Chore no total eligible expenses on advanced milestones

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH CARD 1513](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1513)

### CODE REVIEW

File: app/cypress/e2e/cif/project-revision/undo-changes.cy.js [hunk 1]
```diff
@@ -232,6 +232,32 @@ describe("when undoing, the project revision page", () => {
...
+    cy.findByText(/Quarterly report 1/i).click();
+    cy.get('[aria-label="General Comments"]').clear().type("I will be undone");
+    cy.contains("Changes saved.");
+    cy.findByRole("button", { name: /undo changes/i }).click();
+    cy.findByLabelText(/General Comments/i).should(
+      "have.text",
+      "quarterly report comments 1"
+    );
+
+    //TODO: undo TEIMP agreement once fixture added
```
Comment (dleard) [inline app/cypress/e2e/cif/project-revision/undo-changes.cy.js:247]: TODOs in code tend to get lost or live forever. Can this missing test be added to an existing card in zenhub as part of the acceptance criteria?

File: schema/data/prod/json_schema/milestone.json [hunk 2]
```diff
@@ -103,6 +99,30 @@
             "required": ["maximumAmount"]
           }
         ]
+      },
+      "reportType": {
+        "oneOf": [
```
Comment (dleard) [inline schema/data/prod/json_schema/milestone.json:104]: :ok_hand: Nice

**Review by dleard — CHANGES_REQUESTED** [review]
Looks great. I manually tested & the Total Eligible Expenses field is gone from Advanced milestones. Just a note on TODOs in code.

File: app/cypress/e2e/cif/project-revision/undo-changes.cy.js (hunk 1 above)
Comment (Sepehr-Sobhani) [reply app/cypress/e2e/cif/project-revision/undo-changes.cy.js:247]: Thanks for bringing this up. That part of the code was actually redundant and I added that by mistake but will make a tech debt card for that TODO.

Comment (Sepehr-Sobhani) [reply app/cypress/e2e/cif/project-revision/undo-changes.cy.js]: TODO card: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1568

**Comment by Sepehr-Sobhani** [comment]
> Since this is a user story with points. Should one of the commits be a `feat` so that this change shows up in our changelog?

@dleard I wasn't sure about using feat or chore since it's just hiding a field conditionally and not actually a new feature, but if we consider all user stories a `feat` that makes my life easier to understand when to use feat and when not to use it.

**Review by dleard — APPROVED** [review]

**Pull request merged**